server/handler/query.go

```diff
@@ -8,20 +8,34 @@ import (
 	"net/http"
 
 	"github.com/src-d/gitbase-playground/server/serializer"
+	"gopkg.in/bblfsh/sdk.v1/uast"
+
+	"github.com/go-sql-driver/mysql"
 )
 
 type queryRequest struct {
 	Query string `json:"query"`
 	Limit int    `json:"limit"`
 }
 
-// genericVals returns a slice of interface{}, each one a pointer to a NullString
-func genericVals(nColumns int) []interface{} {
-	columnVals := make([]sql.NullString, nColumns)
-	columnValsPtr := make([]interface{}, nColumns)
-
-	for i := range columnVals {
-		columnValsPtr[i] = &columnVals[i]
+// genericVals returns a slice of interface{}, each one a pointer to the proper
+// type for each column
+func genericVals(colTypes []*sql.ColumnType) []interface{} {
+	columnValsPtr := make([]interface{}, len(colTypes))
+
+	for i, colType := range colTypes {
```

Comment:
should we cover all types correctly? For example I don't see double or float.
https://github.com/go-sql-driver/mysql/blob/master/fields.go#L16

Reply:
You are right, I just added the types I encountered them in gitbase replies, but it's better to make it complete since the beginning. Done in e6deab2